portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase

```diff
@@ -1093,6 +1093,43 @@ definition {
 			userScreenName = "testscreen");
 	}
 
+	@description = "This is a use case for LPS-175323 - FragmentEntryLink table definition references defined incorrectly."
+	@priority = 3
+	test NoChangesCanBeFoundAfterDiscardingHomePageChange {
+		task ("Create Publication and modify Home Page") {
+			JSONPublications.addPublication(publicationName = "Test Publication");
+
+			JSONPublications.selectPublication(publicationName = "Test Publication");
+
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Home",
+				siteName = "Guest");
+
+			PageEditor.addFragment(
+				collectionName = "Basic Components",
+				fragmentName = "Button",
+				targetFragmentName = "Image");
+
+			PageEditor.publish();
+		}
+
+		task ("Discard change and assert no changes were found") {
+			PublicationsNavigator.gotoReviewChanges();
+
+			PublicationsChanges.discardChanges(
+				changeCategory = "Page",
+				changeTitle = "Home");
+
+			PublicationsNavigator.gotoReviewChanges(
+				gotoPublicationsAdmin = "true",
+				publicationName = "Test Publication");
+
+			PublicationsChanges.viewChangesResultNotPresent(
```

Comment:
For the last step, we should assert no changes were found, not just the home change, can you please update the last step to `AssertElementPresent(locator1 = "PublicationsChanges#MESSAGE_NO_CHANGES");`?